### ISSUE TITLE
Non-ASCII Parameters Not Working

### ISSUE THREAD

**Issue by shinkou** [uber/vertica-python#130]
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import json, vertica_python

cnxinfo = {'host': 'verticadb01', 'port': 5433, 'user': 'user1', 'password': 'blahblah', 'database': 'mydb'}

cnx = vertica_python.connect(**cnxinfo)
crsr = cnx.cursor()

## This doesn't work
#crsr.execute(u'SELECT * FROM my_schema.my_table WHERE lang = %s ORDER BY id', (u'Español',))

# This works
crsr.execute(u'SELECT * FROM my_schema.my_table WHERE lang = \'Español\' ORDER BY id')

d = crsr.fetchall()
cnx.close()
print(json.dumps(d))
```

**Comment by BenFeinstein** [comment]
Perhaps it's related to the absence of apostrophes in the first (not working) example?

I would suggest to test whether surrounding the %s placeholder with apostrophes (\'%s\') fixes the broken example.

On Oct 4, 2016 6:20 PM, Chun-Kwong Wong <notifications@github.com> wrote:

#!/usr/bin/env python
# -*- coding: utf-8 -*-
import json, vertica_python

cnxinfo = {'host': 'verticadb01', 'port': 5433, 'user': 'user1', 'password': 'blahblah', 'database': 'mydb'}

cnx = vertica_python.connect(**cnxinfo)
crsr = cnx.cursor()

## This doesn't work
#crsr.execute(u'SELECT * FROM my_schema.my_table WHERE lang = %s ORDER BY id', (u'Espa?ol',))

# This works
crsr.execute(u'SELECT * FROM my_schema.my_table WHERE lang = \'Espa?ol\' ORDER BY id')

d = crsr.fetchall()
cnx.close()
print(json.dumps(d))


-

**Comment by shinkou** [comment]
It works if we replace the 2nd parameter, which is a tuple, with `(u'English',)`.

**Comment by alexjikim** [comment]
thanks for reporting this.
should be fixed as of 0.6.8.
